src/stylesheets/App.css

```diff
@@ -0,0 +1,38 @@
+/* text me one font */
+@import url("https://fonts.googleapis.com/css2?family=Text+Me+One&display=swap");
+
+/* Roboto font */
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+body {
+  background-color: #03191e;
+}
+
+/* Header styles */
+.header-wrapper {
+  margin: 1rem;
+}
+
+.header {
+  font-family: "Text Me One", sans-serif;
+  font-style: normal;
+  font-weight: lighter;
+  font-size: 4rem;
+  letter-spacing: 0.1em;
+  margin: 0;
+  color: #a682ff;
+  -webkit-text-fill-color: black;
+  -webkit-text-stroke-width: 2px;
+  -webkit-text-stroke-color: #a682ff;
+  text-shadow: 5px 5px 10px #d138bf, -1px -1px 10px #d138bf, 3px 2px 6px #d138bf,
+    -1px -2px 5px #d138bf;
```

Comment:
This does a pretty good job, but I've noticed it's not EXACTLY like the Figma sketch. Is there any way to make make the size of the "inner" section of a stroke a little bigger, and apply shadow in there?